controllers/organizations.js

```diff
@@ -57,7 +57,7 @@ module.exports = function(app, config) {
 			
 			pagination.parsePagingRequest(req, function(err, pagingReq) {
 				if (err) return next(err);
-				page = pagingReq.page;
+				page = pagingReq.page - 1; // limit 0,1 in case the page is 1
```

Comment:
Same as before this should have other issue while generating the paging links in response.